Makefile

```diff
@@ -28,7 +28,7 @@ release: \
 	_output/release/bootkube.tar.gz \
 
 check:
-	@find . -name vendor -prune -o -name '*.go' -exec gofmt -s -d {} +
+	@gofmt -s -d $$(find . -name '*.go' -print -o -name vendor -prune); if [ "$$?" = "0" ]; then exit 1; fi
```

Comment:
I tried capturing the output, checking if empty, and then printing the output, instead of invoking `gofmt` twice, but it was more trouble than it was worth to get make/shell to do what I wanted.